dove/src/cmd/init.rs

```diff
@@ -89,6 +89,9 @@ impl Cmd for Init {
         }
 
         if let Some(url) = &self.repository {
+            if !regex::Regex::new(r"^http(s)?://[a-z\d_\-\.]+")?.is_match(&url.to_string()) {
```

Comment:
Do you really need to create this `Regex` every time? 

I'm wondering whether it's a good idea to reinvent URL validation here. Could you maybe use https://docs.rs/url/2.2.2/url/ crate instead?